Dockerfile

```diff
@@ -10,20 +10,14 @@ EXPOSE 80
 
 RUN apt-get update -y && \
     apt-get -y dist-upgrade && \
-    apt-get install -y build-essential \
+    apt-get install -y libz-dev
+RUN apt-get install -y build-essential \
```

Comment:
Can we chain the install above rather than executing a second `RUN` command?  It'll remove a layer from the image.

Reply:
Something like:
```
RUN apt-get update -y && \
    apt-get -y dist-upgrade && \
    apt-get install -y libz-dev \
                       build-essential \
                       curl \
                       git \
                       ldap-utils \
                       tzdata \
    && rm -rf /var/lib/apt/lists/*
```

Reply:
Done

Reply:
On using one layer the OSS build fails.
We are not sure why and would like to postponed it for the future.